uaclient/util.py

```diff
@@ -458,3 +459,20 @@ def deduplicate_arches(arches: List[str]) -> List[str]:
 
 def we_are_currently_root() -> bool:
     return os.getuid() == 0
+
+
+def print_package_list(
+    package_list: List[str],
+):
+    package_names = " ".join(package_list)
```

Comment:
Just a nit, but I don't think we need this variable here, we can the operation directly on the print call